Representable/representable
# added profanity filter

added filter to screen for profanity-- must install the google client library for python which can be done by typing the following into terminal:

`pip install --upgrade google-api-python-client google-auth-httplib2 google-auth-oauthlib`

note that we will also need to install this on the representable server

Testing TODO:

- Install library `pip install --upgrade google-api-python-client google-auth-httplib2 google-auth-oauthlib`
- make map using profane language in any of the descriptions
- check the community_entry table to see if private == TRUE for that entry
- make map using non-profane language in all descriptions
- check the community_entry table to see if private == FALSE for that entry
- (optional, only do if interested) you can fiddle around with the metrics to see if you like what we consider to be profane. i spent a lot of time tweaking the settings myself, but let me know if you're interested; i can send a short Python file with that
